### DISCUSSION TITLE
How to make Circular motion of letters around the center

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#8130] [Q&A]
How to make Circular motion of letters around the center
![image](https://user-images.githubusercontent.com/1198840/150636661-07ac2f8d-00d4-435d-b7e0-281c4bc7283d.png)

**Comment by bigtimebuddy** [comment]
Pixi does not support this feature. There are tools like Adobe Photoshop that will allow you to pre-render a text treatment in 3D. If you need something dynamic, you'll need to come up with a custom solution. Maybe check with Babylon or three.js if they support this?
